### PR TITLE
README files update for Glyphs - Issue 15

### PR DESCRIPTION
**Pull request by goksel**
I updated the readme files to fix issues listed here: https://github.com/SynBioDex/SBOL-visual/issues/15

I believe all six issue are now fixed. 

EXCEPT:
The actual PNG glyphs for 3' overhang and sticky restriction sites STILL need to be created. The readme files were created correctly using the proper file names. Once the PNG files are replaced, this issue can be closed.

### CODE REVIEW

File: Glyphs/FunctionalComponents/complex/README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 SBO:0000253 Non-covalent complex
 
 ## Recommended Glyph and Alternates
-The RECOMMENDED glyph for a complex is a composite of the glyphs for the molecules of comprising the complex.  For example, a protein bound to a simple chemical, a protein bound to a guide RNA, or two proteins bound together:
+The RECOMMENDED glyph for a complex is a composite of the glyphs for the molecules comprising the complex.  For example, a protein bound to a simple chemical, a guide RNA, or another protein (in order:protein-small molecule, protein-guide RNA, protein-protein):
 
-![glyph specification](complex-ps-specification.png)
+![glyph specification](complex-ps-specification.pngs
```
Comment (jakebeal) [inline Glyphs/FunctionalComponents/complex/README.md]: This is now creating a broken link --- I assume the change is accidental?

**Comment by jakebeal** [comment]
I've changed the target from master to a new branch, in order to allow resolution of the last issues before merging into master.

**Review by jakebeal — CHANGES_REQUESTED** [review]

**Review by jakebeal — APPROVED** [review]

**Comment by jakebeal** [comment]
I'm now going to finish the last bits in the ontology-update branch

**Comment by goksel** [comment]
Thanks @jakebeal, I must have introduced the typo as a mistake somehow.

**Pull request merged**